packages/embed/README.md

```diff
@@ -2,7 +2,108 @@
+| medium                 | string      | name of the plugin built on top of the SDK                                                                                 | `"embed-sdk"`                                                 |
+| mediumVersion          | string      | version of the plugin built on top of the SDK                                                                              | `"next"`                                                      |
+| transitiveSearchParams | string[]    | search parameters to be forwarded from host page to form                                                                   | `undefined`                                                   |
+| hideFooter             | boolean     | hide form progress bar and navigation buttons                                                                              | `false`                                                       |
+| hideHeaders            | boolean     | hide header that appears when you have a question group, or a long question                                                | `false`                                                       |
+| opacity                | number      | form background opacity, number from 0 (fully transparent) 100 (fully opaque)                                              | `100`                                                         |
+| disableAutoFocus       | boolean     | disable form auto focus when loaded                                                                                        | `false`                                                       |
+| open                   | string      | open embed based on user action (see below)                                                                                | `undefined`                                                   |
+| openValue              | number      | based on `open` (see below)                                                                                                | `undefined`                                                   |
+| onReady                | function    | fires when the form is loaded                                                                                              | `undefined`                                                   |
+| onSubmit               | function    | fires when user submits the form                                                                                           | `undefined`                                                   |
+| onQuestionChanged      | function    | fires when user navigates between form questions                                                                           | `undefined`                                                   |
+
```

Comment:
👍  nice props table

Reply:
The table would be messy with examples. I want to deploy examples somewhere, but that is yet to be determined.